Mathlib/Data/Matrix/Mul.lean

```diff
@@ -952,10 +961,10 @@ section CommSemiring
 
 variable [CommSemiring α]
 
+@[deprecated mulVec_smul (since := "2025-08-14")]
```

Comment:
Why not `deprecated alias` rather than repeating the statement?

Reply:
The generality and argument order are different, so I figured it was safer this way.